### ISSUE TITLE
Unable to extract metadata from the query. Results won't be editable.

### ISSUE THREAD

**Issue by Spider-Admin** [pawelsalawa/sqlitestudio#4514]
### Details

Table
```
CREATE TABLE IF NOT EXISTS Test (
    ID       INTEGER        CONSTRAINT MyPK PRIMARY KEY AUTOINCREMENT
                            NOT NULL,
    MyText1   VARCHAR (1024) NOT NULL,
    MyText2 VARCHAR (1024) 
);
INSERT INTO Test (ID, MyText1, MyText2) VALUES (1, 'test1', 'test2');
```

Query (working)
```
select ID, MyText2 from Test
```

Query (not working)
```
select ID, 'test' || MyText1  as example, MyText2 from Test
```

In the version from 2022-07-08 I was able to click into the column MyText2, edit it and commit my changes. This does not work anymore in the version from 2022-09-12.

The status windows shows me this 2 messages;
[20:05:32] SQLiteStudio was unable to extract metadata from the query. Results won't be editable.
[20:05:34] Cannot edit this cell. Details: The query execution mechanism had problems with extracting ROWID's properly. This might be a bug in the application. You may want to report this.

### Steps to reproduce
See Details

### Operating system
Windows 10 64 Bit

### SQLiteStudio version
3.4 from 2022-09-12.

**Comment by pawelsalawa** [comment]
Confirmed.

**Comment by pawelsalawa** [comment]
This was fixed in later commits of 3.4.0. I'm not sure when exactly, but it works now as expected.
